components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -20,8 +20,8 @@
 
 @interface MDCTabBarView ()
 
-/** The views representing the items of this tab bar. */
-@property(nonatomic, strong) NSArray<UIView *> *itemViews;
+/** The stack view that contains all tab items. */
+@property(nonatomic) UIStackView *stackView;
```

Comment:
Not sure if this change resolve this issue..

Reply:
I don't see a difference. Perhaps you need to push? I think it's also better to name this for its semantic use (rather than its type).  I think something like this would be better:

```objc
@property(nonnull, nonatomic, strong) UIStackView *containerView;
```

Reply:
Fix applied!